### PR TITLE
Fix: Posts with attached links work again

### LINKED ISSUE #14197: Links and link previews are not distributed
- [x] I have searched open and closed issues for duplicates

### Bug Description
News accounts do not distribute links and link previews to mirror posts if the 2024.06-dev 1564/1564 post update version 1550/1550 is used.

Actual Result:
No link and link preview is distributed

Expected Result:
Link and link preview will be distributed again

### Platform Info

Friendica Version: Stable / 2024.06-dev 1564/1564 Post-Update-Version 1550/1550.

Friendica Source: git

PHP version: 8.1/8.2

SQL version: 10.5

### PR DESCRIPTION
**Pull request by annando**
Fixes #14197

### CODE REVIEW

File: src/Protocol/ActivityPub/Transmitter.php [hunk 1]
```diff
@@ -1871,7 +1868,7 @@ public static function createNote(array $item, bool $api_mode = false): array
 		if (!empty($summary)) {
 			$data['summary'] = $summary;
 		}
-
+
```
Comment (MrPetovan) [inline src/Protocol/ActivityPub/Transmitter.php]: Superfluous whitespace.
```suggestion

```

File: src/Protocol/ActivityPub/Transmitter.php [hunk 2]
```diff
@@ -1898,7 +1895,7 @@ public static function createNote(array $item, bool $api_mode = false): array
 		 * }
 		 */
 
-		if (empty($item['uid']) || !Feature::isEnabled($item['uid'], Feature::EXPLICIT_MENTIONS)) {
+		 if (empty($item['uid']) || !Feature::isEnabled($item['uid'], Feature::EXPLICIT_MENTIONS)) {
```
Comment (MrPetovan) [inline src/Protocol/ActivityPub/Transmitter.php]: Superfluous whitespace.
```suggestion
		if (empty($item['uid']) || !Feature::isEnabled($item['uid'], Feature::EXPLICIT_MENTIONS)) {
```

**Review by MrPetovan — CHANGES_REQUESTED** [review]

File: src/Protocol/ActivityPub/Transmitter.php (hunk 1 above)
Comment (annando) [reply src/Protocol/ActivityPub/Transmitter.php]: Done

File: src/Protocol/ActivityPub/Transmitter.php (hunk 2 above)
Comment (annando) [reply src/Protocol/ActivityPub/Transmitter.php]: Done

**Review by MrPetovan — APPROVED** [review]

**Pull request merged**